source/_components/sensor.filter.markdown

```diff
@@ -120,3 +128,10 @@ To adjust the rate you need to set the window_size. To throttle a sensor down to
 
 This filter is relevant when you have a sensor which produces states at a very high-rate, which you might want to throttle down for storing or visualization purposes. 
 
+### {% linkable_title Time Simple Moving Average %}
+
+The Time SMA filter (`time_simple_moving_average`) is based on the paper [Algorithms for Unevenly Spaced Time Series: Moving Averages and Other Rolling Operators](http://www.eckner.com/papers/Algorithms%20for%20Unevenly%20Spaced%20Time%20Series.pdf) by Andreas Eckner. 
+
+The paper defines three types/versions of the Simple Moving Average (SMA): *last*, *next* and *linear*. Currently only *last* is implemented.
+
+Theta, as described in the paper, is the `window_size` parameter, and can be expressed using time notation (e.g., 00:05 for a five minutes time window).
```

Comment:
When we start adding other types, I think it would be nice to provide a separate `configuration.yaml` snippet for each `filter:`, because from the current layout it's quite difficult to find what you're really interested in (especially as a new user). Just a comment and out of scope for this PR of course.